File: apps/ios/ExpUI/Sources/IssueGraphPopover.swift
import ExpCore
import SwiftUI

/// EXP-1082 (workflow contract): the blocks MINI-GRAPH as a shared ExpUI
/// view — the same inputs the issue list's row-badge sheet draws today
/// (`IssueGraphSheet` / `IssueGraphView` in the app target): the issue's
/// transitive `IssueGraph.Graph`, the issues it names, and a tap-to-open.
/// STUB: renders nothing yet; EXP-1057 fills it by COPYING (not moving) the
/// drawing out of the app target's IssueGraphView, which IssueListView keeps
/// using until then.
public struct IssueGraphPopover: View {
    public let graph: IssueGraph.Graph
    public let issues: [IssueEntity]
    public let onOpenIssue: (String) -> Void

    public init(
        graph: IssueGraph.Graph,
        issues: [IssueEntity],
        onOpenIssue: @escaping (String) -> Void
    ) {
        self.graph = graph
        self.issues = issues
        self.onOpenIssue = onOpenIssue
    }

    public var body: some View {
        EmptyView()
    }
}

File: apps/android/app/src/main/java/com/exponential/app/ui/components/IssueGraphPopover.kt
package com.exponential.app.ui.components

import androidx.compose.foundation.layout.Box
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import com.exponential.app.data.db.IssueEntity
import com.exponential.app.domain.IssueGraph

/**
 * EXP-1082 STUB: THE mini-graph as a popover — one drawing for the blocks
 * badge/rail and the workflow detail. Takes the same inputs [IssueGraphList]
 * draws from today and renders nothing yet. EXP-1057 fills it by COPYING
 * (not moving) the drawing out of `IssueGraphList.kt` /
 * `ui/workflows/WorkflowGraphList.kt`; the old files keep working meanwhile.
 */
@Suppress("UNUSED_PARAMETER")
@Composable
fun IssueGraphPopover(
    graph: IssueGraph.Graph,
    issuesById: Map<String, IssueEntity>,
    onOpenIssue: (String) -> Unit,
    modifier: Modifier = Modifier,
) {
    Box(modifier)
}
